CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# [Please contribute](#please-contribute)
+
+You can really make a difference by:
+
+- [Making an issue](https://help.github.com/articles/creating-an-issue/). A well-described issue helps a lot. (Check the [known issues](https://github.com/search?q=user%3Avitabaks+is%3Aissue+state%3Aopen) first.)
+- [Making a pull request](https://docs.github.com/fr/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) when you find an error in the code.
+
+I will try to help and take every contribution seriously.
+
+This is a great opportunity for me to learn how you use the role and also an opportunity for you to get into the habit of contributing to open-source software.
+
+## [Step by step](#step-by-step)
+
+Here is how you can help, many steps are related to GitHub, not specifically to this project.
+
+### [1. Make an issue.](#1-make-an-issue)
+
+When you spot an issue, [create an issue](https://github.com/vitabaks/postgresql_cluster/issues).
+
+Making the issue helps me and others to find similar problems in the future.
+
+### [2. Fork the project.](#2-fork-the-project)
+
+On the top right side of [the repository on GitHub](https://github.com/vitabaks/postgresql_cluster), click `fork`. This copies everything to your GitHub namespace.
+
+### [3. Make the changes](#3-make-the-changes)
+
+In your GitHub namespace, make the required changes.
+
+I typically do that by cloning the repository (in your namespace) locally:
+
+```
+git clone git@github.com:YOURNAMESPACE/postgresql_cluster.git
+```
+
+Now you can start editing on your laptop.
+
+### [4. Test your changes](#4-optionally-test-your-changes)
+
+Install [make](https://www.gnu.org/software/make/) and [Python3.10](https://www.python.org/) with [venv](https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtual-environments/) :
+
+```
+sudo apt install -y make python3.10 python3.10-venv
```

Comment:
```
gitpod@thomassanso-postgresqlc-9nwq8iqng2j:/workspace/postgresql_cluster$ sudo apt install -y make python3.10 python3.10-venv
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package python3.10-venv is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'python3.10-venv' has no installation candidate
```

Reply:
It seems that the command `sudo apt update` is missing in the current configuration, which is necessary for proper functionality. I will add this command to ensure everything works as intended.

Reply:
Done and tested on gitpod (working)